instant-xml/tests/all.rs

```diff
@@ -27,8 +51,35 @@ fn struct_with_named_fields() {
             string: "test".to_string(),
             number: 1,
         }
-        .to_xml()
+        .to_xml(None)
         .unwrap(),
         "<StructWithNamedFields xmlns=\"URI\" xmlns:bar=\"BAZ\" xmlns:foo=\"BAR\"><flag>true</flag><bar:string>test</bar:string><number xmlns=\"typo\">1</number></StructWithNamedFields>"
     );
 }
+
+#[test]
+fn struct_with_custom_field() {
+    assert_eq!(
+        StructWithCustomField {
+            test: Nested {
```

Comment:
I think we want to ditch the `<test>` tag here and only keep `<Nested>`.

Reply:
done